pkg/centralsensor/caps_list.go

```diff
@@ -74,4 +74,10 @@ const (
 
 	// VirtualMachinesSupported identifies the capability of Central to receive virtual machine requests.
 	VirtualMachinesSupported = "VirtualMachinesSupported"
+
+	// FlattenImageDataOnSensor identifies the capability of Sensor to use the flattened image data model (ImageV2).
+	FlattenImageDataOnSensor SensorCapability = "FlattenImageDataOnSensor"
+
+	// FlattenImageDataOnCentral identifies the capability of Central to use the flattened image data model (ImageV2).
+	FlattenImageDataOnCentral CentralCapability = "FlattenImageDataOnCentral"
```

Comment:
You may not need two separate variables here (depending on usage)

```suggestion
	// FlattenImageData identifies the capability to use the flattened image data model (ImageV2).
	FlattenImageData = "FlattenImageData"
```

Are a few other caps that are set separatly by Central vs. Sensor but share the same 'value' - ie: `ScannerV4Supported`, `ClusterRegistrationSecretSupported`, etc.